### PR TITLE
Hw 5.strings files

### PR DESCRIPTION
**Pull request by ninaprus**
Working with files, reading and writing files

### CODE REVIEW

File: hw_5_strings_file.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#1)Из текстового файла удалить все слова, содержащие от трех до пяти символов,
+# но при этом из каждой строки должно быть удалено
+# только четное количество таких слов.
+
+with open('example.txt') as reader:
+    file = reader.readlines()
```
Comment (katevoskresenska) [inline hw_5_strings_file.py]: It is possible to iterate through lines with:
with open('example.txt') as reader:
for line in reader:

File: hw_5_strings_file.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            if 3 <= len(item) <= 5:
+                counter += 1
+                len_3_5.append(item)
+        #parity check
+        if counter%2 == 0:
+            line2 = ' '.join([item for item in line if item not in len_3_5])
+        else:
+            line2 = ' '.join([item for item in line if item not in len_3_5[0:-1]])
+
+        print(f'{line2}')
```
Comment (katevoskresenska) [inline hw_5_strings_file.py]: Output should be written to the resulting file

File: hw_5_strings_file.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+#2)Текстовый файл содержит записи о телефонах и их владельцах.
+# Переписать в другой файл телефоны тех владельцев,
+# фамилии которых начинаются с букв К и С.
+
+with open('phone_num.txt', 'r') as reader_ph, open('phone_num_new.txt', 'w') as writer_ph:
+    for line in reader_ph:
+        for item in line:
+            if item.isalpha():
```
Comment (katevoskresenska) [inline hw_5_strings_file.py]: I think, this check can be excluded

File: hw_5_strings_file.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    file = reader2.readlines()
+    #find the maximum length of lines
+    len_line = []
+    for line in file:
+        len_line.append(len(line))
+    max_len = max(len_line)
+
+    for line in file:
+        line = line.rstrip('\n')
+        print(line.rjust(max_len))
```
Comment (katevoskresenska) [inline hw_5_strings_file.py]: Output should be written to the resulting file as well

File: hw_5_strings_file.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
+#1)Из текстового файла удалить все слова, содержащие от трех до пяти символов,
```
Comment (katevoskresenska) [inline hw_5_strings_file.py:1]: It is a general comment for all the tasks - the code should be represented with functions

Comment (ninaprus) [reply hw_5_strings_file.py:1]: Thanks. I did (tried)

File: hw_5_strings_file.py [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+#2)Текстовый файл содержит записи о телефонах и их владельцах.
+# Переписать в другой файл телефоны тех владельцев,
+# фамилии которых начинаются с букв К и С.
+
+def get_phone_CK(my_file):
+    """Function creates a new text file that contains telephones
+     last names starting with the letters K and C."""
+
+    with open(my_file) as reader_ph, open('phone_num_new.txt', 'w') as writer_ph:
```
Comment (mmykhailenko) [inline hw_5_strings_file.py:38]: check if the file exists before read it

**Review by mmykhailenko — APPROVED** [review]

**Pull request merged**